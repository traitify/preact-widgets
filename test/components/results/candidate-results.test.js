import Component from "components/results/candidate-results";
import ComponentHandler from "support/component-handler";

jest.mock("components/results/candidate-results/personality-archetype", () => (() => (<div className="mock">Personality Archetype</div>)));
jest.mock("components/results/candidate-results/personality-details", () => (() => (<div className="mock">Personality Details</div>)));
jest.mock("components/results/candidate-results/personality-dimensions", () => (() => (<div className="mock">Personality Dimensions</div>)));
jest.mock("lib/with-traitify", () => ((value) => value));

describe("CandidateResults", () => {
  it("renders results", () => {
    const component = new ComponentHandler(<Component />);

    expect(component.tree).toMatchSnapshot();
  });
});
