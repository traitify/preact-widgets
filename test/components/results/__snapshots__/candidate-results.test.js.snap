// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CandidateResults renders results 1`] = `
<section>
  <div
    className="mock"
  >
    Personality Archetype
  </div>
  <div
    className="mock"
  >
    Personality Dimensions
  </div>
  <div
    className="mock"
  >
    Personality Details
  </div>
  <div
    className="mock"
  >
    Personality Traits
  </div>
</section>
`;
